gandalf.js

```diff
@@ -7,111 +7,105 @@
  * See README.md for details.
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?